iroh-test/README.md

```diff
@@ -0,0 +1,4 @@
+# iroh-test
+
+This crate contains testing support code for iroh.  It is only used
+for running the iroh test suite.
```

Comment:
should have the license footer as in the other readmes

Reply:
also needs license files

Reply:
> also needs license files

We only have license files in the workspace though?

Reply:
hmm, you are right, then lets leave it as is for now